{
  "commits": {
    "buggyBuild": {
      "repoName": "ldbc/ldbc_snb_implementations",
      "branchName": "master",
      "sha": "98c4d89c9bf682c0fed5fb9aec881ab3795914c4",
      "url": "http://github.com/ldbc/ldbc_snb_implementations/commit/98c4d89c9bf682c0fed5fb9aec881ab3795914c4",
      "date": "Aug 13, 2018 12:16:28 AM"
    }
  },
  "tests": {
    "failingModule": "/root/workspace/ldbc/ldbc_snb_implementations/415222188/cypher",
    "overallMetrics": {
      "numberRunning": 52,
      "numberPassing": 2,
      "numberFailing": 0,
      "numberErroring": 50,
      "numberSkipping": 1,
      "failures": [
        {
          "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
          "isError": true,
          "occurrences": 25
        },
        {
          "failureName": "java.lang.NullPointerException",
          "isError": true,
          "occurrences": 25
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "cypher.CypherBiTest",
        "numberRunning": 50,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 50,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery17",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery13",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery8",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery13",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery18",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery11",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery23",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery5",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery22",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery9",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery25",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery2",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery4",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery24",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery15",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery20",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery16",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery12",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery10",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery19",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery3",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery21",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery20",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery10",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery25",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery17",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery8",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery7",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery9",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery21",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery6",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery14",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery18",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery24",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery23",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery16",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery6",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery1",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery2",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery19",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery4",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery1",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery5",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery12",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery3",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery15",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery7",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery11",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery14",
        "failureName": "java.lang.NullPointerException",
        "isError": true
      },
      {
        "testClass": "cypher.CypherBiTest",
        "testMethod": "testBiQuery22",
        "failureName": "org.neo4j.driver.v1.exceptions.ServiceUnavailableException",
        "detail": "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.",
        "isError": true
      }
    ]
  },
  "repository": {
    "name": "ldbc/ldbc_snb_implementations",
    "githubId": 20736043,
    "url": "https://github.com/ldbc/ldbc_snb_implementations",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 415222188,
      "url": "http://travis-ci.org/ldbc/ldbc_snb_implementations/builds/415222188",
      "date": "Aug 13, 2018 12:20:10 AM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 13, 2018 12:20:21 AM",
    "reproductionDateEnd": "Aug 13, 2018 12:27:53 AM",
    "totalDuration": 448,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 7,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          7
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 207,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          207
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 65,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          65
        ]
      }
    },
    "projectRootPomPath": "./workspace/ldbc/ldbc_snb_implementations/415222188/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "projectMetrics": {
    "numberModules": 4,
    "numberSourceFiles": 38,
    "numberTestFiles": 8,
    "numberLibrariesFailingModule": 2
  },
  "type": "only_fail"
}